src/main/java/Duke.java

```diff
@@ -1,10 +1,214 @@
+import duke.tasks.*;
+import duke.exceptions.*;
```

Comment:
Consider the use of import all statements, even if you require everything in a package, it would be good practice to specify them as individual imports.